parm/config/gfs/config.resources.HERA

```diff
@@ -30,6 +34,13 @@ case ${step} in
     export tasks_per_node=$(( max_tasks_per_node / threads_per_task ))
     ;;
 
+  "upp")
+    if [[ "${CASE}" == "C768" ]]; then
```

Comment:
I imagine higher resolutions will need to do this as well (if not spread it even more thin). Recommend applying to all higher resolutions for now until a higher resolution requires even more spreading.
```suggestion
    if (( "${CASE:1}" >= 768 )); then
```